schema/src/CreditClassInfo.yaml

```diff
@@ -0,0 +1,90 @@
+id: https://framework.regen.network/schema/CreditClassInfo
+name: CreditClassInfo
+prefixes:
+  linkml: https://w3id.org/linkml/
+  rfs: https://framework.regen.network/schema/
+  schema: http://schema.org/
+  qudt: http://qudt.org/schema/qudt/
+  unit: http://qudt.org/vocab/unit/
+  xsd: http://www.w3.org/2001/XMLSchema#
+imports:
+  - taxonomy
+  - SDG
+  - ProjectBenefit
+  - PrimaryImpact
```

Comment:
I created these classes in separate yaml files because these classes might be used outside of credit classes. In each of these files I also defined slots that can be used to reference these classes. It's possible that a credit class might have multiple slots referencing the same class for different semantic purposes, in which case each class can define these specific slots themselves.